Gray_Code/Gray_Code.cpp

```diff
@@ -0,0 +1,53 @@
+//C++ program to generate the Gray Code for any input bit length
+
+#include<bits/stdc++.h>
+#define pb push_back
+using namespace std;
+
+// Function to generate the Gray Code sequence
+void Gray(int n){
+    //If input size is less than or equal to 0
+    if(n <= 0){
+        cout << "Invalid input";
+        return;
+    }
+
+    vector<string> v;
+    v.pb("0");
+    v.pb("1");
+
+    for(int i = 2; i < (1 << n); i <<= 1){
+    	for(int j = i - 1; j >= 0; j--)
+    	    v.pb(v[j]);
+        
+    	for(int j = 0; j < i; j++)
+            v[j] = "0" + v[j];
+        
+    	for(int j = i; j < 2 * i; j++)
+    	    v[j] = "1" + v[j];
+    }
+
+    cout << "Gray Code for bit length = " << n << " : " << endl;
+
+    for(int i = 0; i < v.size(); i++)
+    	cout << v[i] << endl;
```

Comment:
Please add 4 space indentation here.

Reply:
![Capture](https://user-images.githubusercontent.com/46227193/80403631-89d7e980-88dd-11ea-9d81-c20ae3307e22.PNG)
This is the screenshot of the file that I made.In the file I have done 4 space indentation but I dont understand why its not showing in the files changed seection.Also I have ensured the indentation in github as "Spaces" and "4".

Reply:
@singh-shreya6 I request you to kindly open the file in my branch and review it